### DISCUSSION TITLE
Method for disabling tracking on localhost?

### DISCUSSION THREAD

**Discussion by notflip** [johannschopplich/nuxt-gtag#20] [Q&A] [answered]
Is there a way to quickly disable this plugin for localhost? Otherwise it's working great! The first gtag implementation that actually shows the correct page titles. Thank you!

**Comment by azharumar** [comment]
What I did was to keep the `NUXT_PUBLIC_GTAG_ID=""` on env file as blank.

**Comment by johannschopplich** [comment, marked as answer]
You can unset the Gtag ID during development:

```ts

// `nuxt.config.ts`
export default defineNuxtConfig({
  modules: ["nuxt-gtag"],

  $development: {
    gtag: {
      id: undefined,
    },
  },

  gtag: {
    id: "G-XXXXXXXXXX",
  },
});
```
